src/frontend/src/lib/components/core/Background.svelte

```diff
@@ -0,0 +1,14 @@
+<script lang="ts">
+	import background from '$lib/assets/background.svg';
+	import Img from '$lib/components/ui/Img.svelte';
+	import { i18n } from '$lib/stores/i18n.store';
+	import { replaceOisyPlaceholders } from '$lib/utils/i18n.utils';
+</script>
+
+<div class="fixed inset-0 z-[-1] overflow-hidden bg-white">
+	<Img
+		src={background}
+		alt={replaceOisyPlaceholders($i18n.core.alt.background)}
+		extraClass="min-w-[1280px] absolute left-1/2 top-0 transform -translate-x-1/2 sm:block hidden"
```

Comment:
Not sure if I should use `srcset` to hide the image or the condition `sm:block hidden` is ok

Reply:
`srcset` in my opinion. The wallet is potentially used everything in the word and in some countries there are more phones than desktop, plus where data are not unlimited - i.e. those consumers are probably happy not to have to download an unused asset image. Makes sense?

Reply:
right! otherwise, they would have to load the background image always.... check last commit please